hyperledger-student-management.yaml

```diff
@@ -0,0 +1,232 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Student API for UC4's hyperledger component."
+  version: "0.3.3"
+  title: "UC4"
+servers:
+  - url: http://localhost:9000/hyperledger-student-management
+tags:
+- name: "Student Management"
+  description: "Everything about the student"
+  externalDocs:
+    description: "Find out more"
+    url: "https://github.com/upb-uc4"
+paths:
+  /users/students:
+    post:
+      tags:
+      - "Student Management"
+      summary: "Add a new student to the database"
+      description: "Adds a new student to the database after verification."
+      operationId: "addStudent"
+      security:
+        - uc4_auth: [] 
+      requestBody:
+        required: true
+        description: "Student object that needs to be added to the database"
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Student'
+      responses:
```

Comment:
Right, the errors will definitely have to conform to the new format. We will have to take a look at the errors anyways, since hyperledger does not allow us to return http error codes.